frame/lending/src/lib.rs

```diff
@@ -846,41 +857,98 @@ pub mod pallet {
 			Ok(should_warn)
 		}
 
+		/// Initiate liquidation of individual position for particular borrower within mentioned
+		/// market. Returns 'Ok(())' in the case of successful initiation, 'Err(DispatchError)' in
+		/// the opposite case.
+		/// - `liquidator` : Liquidator's account id.
+		/// - `market_pair` : Index and configuration of the market from which tokens were borrowed.
+		/// - `account` : Borrower's account id whose debt are going to be liquidated.
+		fn liquidate_position(
+			liquidator: &<Self as DeFiEngine>::AccountId,
+			market_pair: &(&<Self as Lending>::MarketId, MarketConfigOf<T>),
+			borrow_asset: <T as DeFiComposableConfig>::MayBeAssetId,
+			account: &<Self as DeFiEngine>::AccountId,
+		) -> Result<(), DispatchError> {
+			let (market_id, market) = market_pair;
```

Comment:
not sure, usually there is no such rust patter with pair passed and then split. can do differently?

Reply:
We can use structure with two public fields.... I need this pair because of: https://github.com/ComposableFi/composable/pull/1065#discussion_r884573202